pkg/hook/register.go

```diff
@@ -23,4 +23,28 @@ func RegisterMethodHook(methodName, receiverType, hookName string) {
 
 func RegisterDefaultHooks() {
```

Comment:
Minor nitpick: Would be nice to sort the hooks by `methodName`. This would be a cheap QoL fix to improve readability/reviewability.